cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/Transactions.java

```diff
@@ -44,6 +50,21 @@ public static void invalidateQuietly(TransactionSystemClient txClient, Transacti
     }
   }
 
+  /**
+   * Handy method to create {@link TransactionExecutor} (See TEPHRA-71).
+   */
+  public static TransactionExecutor createTransactionExecutor(TransactionExecutorFactory factory,
+                                                              Iterable<? extends TransactionAware> txAwares) {
+    return factory.createExecutor(Iterables.transform(txAwares, Functions.<TransactionAware>identity()));
```

Comment:
oh, good. If it had been a comment on this line, saying, we have to do this identity transform until TEPHRA-71 is fixed, it would have be more obvious.